testproject/tests/test_first_step_authentication.py

```diff
@@ -89,16 +97,4 @@ def test_login_wrong_password(active_user):
         format="json",
     )
     assert response.status_code == 400
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too